t/app/controller/waste_peterborough.t

```diff
@@ -848,6 +864,18 @@ FixMyStreet::override_config {
             is $report->get_extra_field_value('ITEM_05'), '';
         };
 
+        subtest '?type=bulky redirect after bulky booking made' => sub {
+            $mech->get_ok('/waste?type=bulky');
+            $mech->content_contains( 'What is your address?',
+                'user on address page' );
+            $mech->submit_form_ok(
+                { with_fields => { postcode => 'PE1 3NA' } } );
+            $mech->submit_form_ok(
+                { with_fields => { address => 'PE1 3NA:100090215480' } } );
+            is $mech->uri,
+                'http://localhost/waste/PE1%203NA:100090215480',
+                'Redirected to waste base page';
```

Comment:
Would be good to test that the "You have a pending bulky collection." message appears on the page at this point.

Reply:
Rather than removing the test, better to update it with the wording that you expect to appear:

```perl
$mech->content_contains('You need to sign in to see the full details of this property.');
```

Reply:
This fails for me, I am assuming because the `NOT c.user_exists` check in the bin days template is false. I am not sure how to get a good bit of text to display without rewriting more of the template so it uses actual values rather than placeholders.

Reply:
Ah yes I see. I guess you could log out at that point in the test? Might have to log back in again if subsequent tests require it.